src/Nethermind/Nethermind.Config/NetworkNode.cs

```diff
@@ -58,5 +58,7 @@ public NetworkNode(PublicKey publicKey, string ip, int port, long reputation = 0
         public string Host => _enode.HostIp.ToString();
         public int Port => _enode.Port;
         public long Reputation { get; set; }
+
+        public NetworkNode(Enode enode) : this(enode.ToString()){}
```

Comment:
This is not what I meant
```suggestion
        public NetworkNode(Enode enode) 
        {
          _enode = enode;
        }
```

also put this next to the other constructor